examples/features/example_difference_of_normals.cpp

```diff
@@ -184,7 +184,8 @@ int main (int argc, char *argv[])
 	range_cond->addComparison (pcl::FieldComparison<PointOutT>::ConstPtr (new
 	pcl::FieldComparison<PointOutT> ("curvature", pcl::ComparisonOps::GT, threshold)));
 	// build the filter
-	pcl::ConditionalRemoval<PointOutT> condrem (range_cond);
+    pcl::ConditionalRemoval<PointOutT> condrem;
+    condrem.setCondition(range_cond);
```

Comment:
Can you please follow the [style guide](http://pointclouds.org/documentation/advanced/pcl_style_guide.php)?

There is an extra space at the beginning of these two lines.
There should be a space before the opening parenthesis.